scripts/components/LviCartItem.ts

```diff
@@ -49,10 +51,34 @@ export default class LviCartItem extends LviCartItemDesign {
         });
     }
     get productPrice(): string {
-        return this.lblProductPrice.text;
+        return this.tvPrice.text;
     }
     set productPrice(value: string) {
-        this.lblProductPrice.text = value;
+        const discountExists = !!this.tvDiscount.text;
+        let attributeString = new AttributedString({
```

Comment:
We should use const if the variable will not change in the next lines.

Reply:
DONE.